File: csharp/Game/Character.cs
namespace Game
{
    public class Character
    {
        internal readonly int armorClass;
        internal readonly int damageDealt;

        public string Race { get; internal set; }
        internal int Force { get; set; }

        public Character(int armorClass, int weaponDamage, string race, int force)
        {
            this.armorClass = armorClass;
            this.damageDealt = weaponDamage;
            Race = race;
            Force = force;
        }
    }
}

File: kotlin/src/main/kotlin/game/Character.kt
package game

open class Character constructor(
    armorClass: Int,
    weaponDamage: Int,
    race: String,
    force: Int
) {
    internal val armorClass: Int = armorClass
    internal val damageDealt: Int = weaponDamage

    var race: String = race
        internal set
    internal var force: Int = force
}